### PR TITLE
Help command

### PR DESCRIPTION
**Pull request by FireGamingYT**
A simple help command displaying all the bot's commands

### CODE REVIEW

File: commands/help.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const util = require('../lib/util.js');
+
+const command = {};
+
+command.command = async (message, args, database, bot) => {
+ const embed = new Discord.MessageEmbed()
+   .setColor(0x43b581)
+   .setAuthor(`Help menu | prefix: !`)
```
Comment (JulianVennen) [inline commands/help.js:8]: The prefix should be retrieved from the config.json file

File: commands/help.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+     { name: "Syntax", value: "`help`\n`article {keyword}`\n`ban {user} {duration} {reason}`\n`helpcenter {zendesk subdomain}`\n`import`\n`ip {require/forbid/off} {channel}`\n`ipcooldown {duration}`\n`kick {user} {reason}`\n`logchannel {channel}`\n`moderations {user}`\n`modroles {add/remove/list} {role}`\n`mute {user} {duration} {reason}`", inline: false},
+     { name: "Syntax", value: "`mutedrole {role}`\n`pardon {case}`\n`ping`\n`playlist {yt playlist link}`\n`punish {strikes} {punishment}`\n`softban {user} {reason}`\n`strike {amount} {user} {reason}`\n`tutorial/video {keyword}`\n`unban {user}`\n`unmute {user}`\n`userinfo/check {user}`", inline: true}
```
Comment (JulianVennen) [inline commands/help.js:15]: This is a very long message. maybe we should link to the commands.md in the GitHub repo instead or only list the commands and show a better description when doing !help commandName

File: commands/help.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+ );
+message.channel.send(embed)
```
Comment (JulianVennen) [inline commands/help.js:18]: missing indent and semicolon (line 2 and 20 are also missing semicolons)

**Comment by FireGamingYT** [comment]
> I'll add descriptions for the commands with e.g. ",help strike" in a new pull request if you want

Sure, let me know if you want me to do anything tho!

**Pull request merged**